nvtabular/ops/categorify.py

```diff
@@ -477,6 +477,27 @@ def _get_dtypes(self):
     fit.__doc__ = StatOperator.fit.__doc__
     fit_finalize.__doc__ = StatOperator.fit_finalize.__doc__
 
+    def compute_output_schema(self, input_schema: Schema, col_selector: ColumnSelector) -> Schema:
+        if not col_selector:
+            col_selector = ColumnSelector(input_schema.column_names)
+        col_selector = self.output_column_names(col_selector)
+        for column_name in col_selector.names:
+            if column_name not in input_schema.column_schemas:
+                input_schema += Schema([column_name])
+        return super().compute_output_schema(input_schema, col_selector)
```

Comment:
Is there something different about this `compute_output_schema` method, or is this left over from before the other refactoring PR that removed the dupes?

Reply:
this was a dupe